cmd/vulndash/cmd/root.go

```diff
@@ -106,6 +108,14 @@ func init() {
 		fmt.Sprintf("the logging verbosity, either %s", log.LevelNames()),
 	)
 
+	rootCmd.PersistentFlags().StringVar(
+		&opts.registryHostname,
+		registryHostnameFlag,
+		"",
+		"the registry hostname for where the images are located to use to filter "+
+			"when getting the reports, ie. gcr.io, us.gcr.io, asia.gcr.io or eu.gcr.io",
```

Comment:
If the only allowed values are `gcr.io`, `us.gcr.io`, `asia.gcr.io` or `eu.gcr.io`, would we add a check for the flag that one of those matches?

Reply:
done @saschagrunert. PTAL